CITi-UFPE/CITiMarco
# Feature/when input part 2

## What I did
- Add code to auto update final date on the date picker and prevent error message from breaking layout
## How to test
- Run the project, and on http://localhost:3000, check if the final date auto updates when changing the initial date
